packages/core/src/core/themes/day.scss

```diff
@@ -1,79 +1,84 @@
+@use 'sass:list';
 @use '../tokens' as castor;
 
-@mixin day() {
-  @include castor.tokens();
+// $type?: class | raw;
+@mixin day($type1: null, $type2: null) {
```

Comment:
Mixin seems to be repeated twice now, possibly more in the future if more themes to be added. Also, does not really help for whom is creating custom themes. Any possibility to have a build function that outputs the required mixin?

Reply:
I'll see what I can do.

Reply:
Unfortunately Sass doesn't support higher order functions (unless they're not anonymous, and it reads poorly).
However I've hoisted the theme logic to its own mixin, making it reusable across theme mixins.

It's still one line more than before, but overall not too bad.
Let me know what you think.

https://github.com/onfido/castor/commit/896eaca808ae5de1146e3651b4039b67e67d2414

Reply:
Looks great, merge! 😏